openai/api_resources/abstract/api_resource.py

```diff
@@ -49,12 +49,12 @@ def instance_url(self, operation=None):
         api_version = self.api_version or openai.api_version
         extn = quote_plus(id)
 
-        if self.typed_api_type == ApiType.AZURE:
+        if self.typed_api_type in (ApiType.AZURE, ApiType.AZURE_AD):
```

Comment:
Was AZURE_AD already defined? I don't see this being added in this PR.

Reply:
It is being added here:
https://github.com/openai/openai-python/blob/e457edc55a6fac37f9c4c0fa51ce0b88862b34f2/openai/util.py#L31